### PR TITLE
Objects Automated Test - LPS-156806 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-156806

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro [hunk 1]
```diff
@@ -186,12 +186,46 @@ definition {
...
+	macro clickOnBackInFilter {
+		Click(locator1 = "ObjectCustomViews#BACK_FILTER");
+	}
+
+	macro clickOnFilterOnEntries {
+		Click(
+			key_buttonName = "Filter",
+			locator1 = "ObjectCustomViews#DROPDOWN_BUTTON_ON_OBJECT_VIEW");
+	}
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro:198]: In the future, we probably can avoid macros like this if we're using tasks to explain what the test is doing.

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectCustomViews.path [hunk 2]
```diff
@@ -245,6 +245,61 @@
...
+	<tr>
+		<td>LABEL_SECTION_FILTER_ENTRY</td>
+		<td>//*[@class='tbar-section']//*[@class='label-section' and contains(.,'${key_filter}')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>BUTTON_RESET_FILTERS</td>
+		<td>//button[contains(@class,'btn btn-unstyled') and contains(text(),'Reset Filters')]</td>
+		<td></td>
+	</tr>
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectCustomViews.path:302]: Hey @andreefarias, I am a little concerned with the sheer amount of new paths that were added for this test. Are you checking existing paths, like in `portal-web/test/functional/com/liferay/portalweb/paths/pathlib`?

When it comes to creating new paths, there is a balance between being flexible to cover multiple use cases vs being specific enough to find the right element on the page. When we refactor, I'll enforce certain standards to add new paths but for now, it's fine.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase [hunk 3]
```diff
@@ -865,6 +865,186 @@ definition {
...
+
+		task ("Change filter by Modified Date") {
+			ObjectCustomViews.clickOnBackInFilter();
+
+			Type(
+				locator1 = "ObjectCustomViews#SEARCH_DROPDOWN_MENU_FILTER",
+				value1 = "Modified Date");
+
+			ObjectCustomViews.defineDateModifiedDateFilter(fromModifiedDate = "01012022");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase:1023]: @andreefarias I see we're changing the filters there to `Created Date` and `Modified Date` but it doesn't look like the test is actually using these to filter the entries. What is the test covering here?

**Review by timpak — COMMENTED** [review]
Hey @andreefarias, can you take a look at my last comment and get back to me? The other ones don't need any action from you but it would be good to consider for the future.

**Comment by andreefarias** [comment]
Hey @timpak,

This ticket is blocked by: [LPS-163011](https://issues.liferay.com/browse/LPS-163011)

**Comment by timpak** [comment]
@andreefarias I'm going to close this PR for now, since it's blocked. Can you do the following:
- move the ticket to `Open` and link the bug ticket as `Blocked by`
- Change the `Test Status` on the test map to `Blocked by Dev` and link the ticket in the comment?

You can re-send once the bug is fixed. Thanks!

**Pull request closed without merging**